docs/guides/applications/configuration-management/ansible/using-the-linode-ansible-collection-to-deploy-infrastructure/index.md

```diff
@@ -1,143 +1,180 @@
+
+        chmod 600 .vault-pass
+
+    {{< caution >}}
+Do not check this file into version control. If this file is located in a Git repository, add it to your [.gitignore file](https://git-scm.com/docs/gitignore).
+{{< /caution >}}
+
+### Create an Ansible Configuration File
+
+Create an Ansible configuration file called `ansible.cfg` with a text editor of your choice. Copy this snippet into the file:
 
 {{< file "~/development/ansible.cfg">}}
-    [defaults]
-    VAULT_PASSWORD_FILE = ./vault-pass
+[defaults]
+VAULT_PASSWORD_FILE = ./vault-pass
 {{< /file >}}
 
-`VAULT_PASSWORD_FILE = ./vault-pass` is used to specify a Vault password file to use whenever Ansible Vault requires a password. Ansible Vault offers several options for password management. To learn more password management, read Ansible's [Providing Vault Passwords Documentation](https://docs.ansible.com/ansible/latest/user_guide/vault.html#providing-vault-passwords) documentation.
+These lines specify the location of your password file.
 
-### Configuration Ansible Vault and Encrypting Variables
+### Encrypt Variables with Ansible Vault
 
-1. In keeping in line with best practices on Ansible, create a directory to store variable files which will contain data encrypted by Ansible Vault, in this case the `~/development/group_vars/example_group/` directory:
+1.  Create a directory to store variable files used with your [Ansible playbooks](/docs/guides/getting-started-with-ansible/#what-is-ansible):
 
         mkdir -p ~/development/group_vars/example_group/
 
-1. Decide on a secure Root Password for any infrastructure that will be deployed using the Linode Collection, then, use ansible-vault to encrypt the string with the following syntax, replacing `MySecureRootPassword` with your own strong and unique password:
+1.  Make a new empty text file called `vars` in this directory. In the next steps, your encrypted API token and root password are stored in this file:
+
+        touch ~/development/group_vars/example_group/vars
+
+1.  Generate a unique, complex new password (for example, by using a password manager) that should be used as the root password for new compute instances created with the Linode Ansible collection. This should be different from the Ansible Vault password specified in the `.vault-pass` file. 
 
-        ansible-vault encrypt_string 'MySecureRootPassword' --name 'password'
+1. Use the following `ansible-vault encrypt_string` command to encrypt the new root password, replacing `MySecureRootPassword` with your password. Because this command is run from inside your `~/development` directory, the Ansible Vault password in your `.vault-pass` file is used to perform the encryption:
 
-    You will be asked to enter a secure vault password to decrypt the variable with later. Enter this password now.
+        ansible-vault encrypt_string 'MySecureRootPassword' --name 'password' | tee -a group_vars/example_group/vars
```

Comment:
Incorporating the `tee` command to copy the encrypted string into the vars file, to help avoid situations where the user makes a copy/paste error between the terminal output and the file. (This actually happened to me on my first run through and was hard to debug 😿 )